pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -156,24 +156,72 @@ void *initObjToJSON(void) {
         (PyTypeObject *)PyObject_GetAttrString(mod_decimal, "Decimal");
     Py_DECREF(mod_decimal);
 
+    if (type_decimal == NULL) {
+        return NULL;
+    }
+
     PyDateTime_IMPORT;
 
     mod_pandas = PyImport_ImportModule("pandas");
-    if (mod_pandas) {
-        cls_dataframe =
-            (PyTypeObject *)PyObject_GetAttrString(mod_pandas, "DataFrame");
-        cls_index = (PyTypeObject *)PyObject_GetAttrString(mod_pandas, "Index");
-        cls_series =
-            (PyTypeObject *)PyObject_GetAttrString(mod_pandas, "Series");
-        cls_timedelta = PyObject_GetAttrString(mod_pandas, "Timedelta");
+    if (mod_pandas == NULL) {
```

Comment:
this should really never happen right?

Reply:
I would think unlikely. Being pedantic for the sake of consistency